resources/uk/gov/hmcts/gradle/init.gradle

```diff
@@ -24,10 +24,6 @@ allprojects {
   task printVersionInit {
     doLast { println project.version }
   }
-
-  task javaVersion {
```

Comment:
Turns out this isn't useful as we need to run on Java 17 to run gradle at all if you have a plugin that requires it.
Could run the gradle jvm on 17 just for this check but that may fail on old gradle versions